### ISSUE TITLE
integrate DynODE into new 4+ repo structure

### ISSUE THREAD

**Issue by arik-shurygin** [CDCgov/DynODE#292]
as discussed in internal communications this repo is going to undergo a reorganization to separate functionality across 3 different repositories.

Repo 1: [DynODE](https://github.com/CDCgov/DynODE) (this one)
- Contains the framework to build respiratory compartment models (currently within `src/resp_ode`)

Repo 2: DynODE-Projects (not yet created)
- Contains published projects so outside users can load and examine the models behind major efforts that DynODE contributed to.

Repo 3: [scenarios-hpc-azure](https://github.com/CDCgov/scenarios-hpc-azure)
- Contains code used to launch experiments (currently found in `src/mechanistic_azure`). This repository will come with scripts that attach to the users `.venv` shell and allow for easy command line launching of jobs onto Azure.

Repos 4+: experiment-specific-repos (cdcent)
- Contains internal experiments which are currently found in the `exp/` folder. Remove this folder from DynODE and separate the experiments into their own repositories. Those repositories may depend on DynODE and use the scenarios-hpc-azure repo to launch the experiments onto azure.

In short after all of these repos are in a running state, the `exp/` and `src/mechanistic_azure` folders should be removed, and we should also modify the packaging and dependencies within `pyproject.toml` to reflect the new state of necessary libraries.

There should also be a public facing example experiment on CDCgov to show external users how to setup their experiments and launch them on their own high performance compute platforms
